overview.md

```diff
@@ -137,6 +138,9 @@ root catalog might be a sub-catalog of someone else's structure. The goal is for
 information and links they want to, while also encouraging a natural web of information to arise as Catalogs and Items are
 linked to across the web.
 
+*NOTE: The above examples all say Catalog, but those can all be Collections as well, as it has all the fields necessary to 
```

Comment:
Maybe put at the top where it starts to get important to say this? May reduce confusion...